lib/active_merchant/billing/credit_card_methods.rb

```diff
@@ -103,37 +128,18 @@ def valid_number?(number)
             valid_checksum?(number)
         end
 
-        # Regular expressions for the known card companies.
-        #
-        # References:
-        # - http://en.wikipedia.org/wiki/Credit_card_number
-        # - http://www.barclaycardbusiness.co.uk/information_zone/processing/bin_rules.html
         def card_companies
-          CARD_COMPANIES
+          CARD_COMPANIES.keys
```

Comment:
Isn't that a breaking change? Someone might be using this method.

Reply:
It's possible. If we're concerned, I can tweak this by adding a parallel structure that returns the regexes. But the regexes will be (honestly already are) wrong, so I'd personally prefer we just loudly note it's a breaking change in the release notes.

Reply:
That works for me!